{"author":{"id":"4a66333145a8be3a1fab83bfb67acb16b49eec9bba93a1d3ada9ca43625279f7"},"ops":[{"type":5,"author":{"id":"4a66333145a8be3a1fab83bfb67acb16b49eec9bba93a1d3ada9ca43625279f7"},"timestamp":1557166470,"nonce":"erG8tCodFWPeR6G+R9ASf9B2dgg=","added":["enhancement"],"removed":null},{"type":4,"author":{"id":"4a66333145a8be3a1fab83bfb67acb16b49eec9bba93a1d3ada9ca43625279f7"},"timestamp":1557522988,"nonce":"VHknKxnVmEPbgABU8nauvQm23ww=","status":2},{"type":3,"author":{"id":"4a66333145a8be3a1fab83bfb67acb16b49eec9bba93a1d3ada9ca43625279f7"},"timestamp":1557524183,"nonce":"zx6nsG7MuCDYWeUQSfshjfrmFDo=","message":"A https://www.pythonanywhere.com/whitelist/ az overpasst explicite engedi, de mást nem. :( Amúgy menne a dolog.","files":null},{"type":3,"author":{"id":"4a66333145a8be3a1fab83bfb67acb16b49eec9bba93a1d3ada9ca43625279f7"},"timestamp":1557526105,"nonce":"gs9c6CcP2txkckZjDsHlzFUV/XI=","message":"Befrissítettem az összes referenciát kézzel (beleértve a fent emlegetett kettőt), illetve beraktam alá a cache-t, szóval hibát nem fog adni ha csak úgy kattingat a user. Ha pedig ténylegesen le akarna tölteni valamit, arra hiba lesz, de legalább a régi lista nem veszik el. Egyelőre legyen az, hogy kommentelj ide ilyen esetben.","files":null}]}